examples/src/rescue_raps/prover.rs

```diff
@@ -10,7 +10,10 @@ use super::{
 
 // RESCUE PROVER
 // ================================================================================================
-
+/// This example constructs a proof for correct execution of
+/// 2 hash chains simulateneously.
```

Comment:
nit: is there a reason to split this sentence into 2 lines?

Reply:
and there's a typo
```suggestion
/// This example constructs a proof for correct execution of
/// 2 hash chains simultaneously.
```